### ISSUE TITLE
Installing node package on windows - incorrect configuration?

### ISSUE THREAD

**Issue by mpdaugherty** [asg017/sqlite-vec#114]
I'm attempting to install the `sqlite-vec` node package on windows, but it appears that the optional dependency `sqlite-vec-windows-x64` is configured to require os "windows" vs. "win32", so it won't install.

```
PS C:\Users\Michael\Documents\TestCode> npm i sqlite-vec-windows-x64
Debugger attached.
npm ERR! code EBADPLATFORM
npm ERR! notsup Unsupported platform for sqlite-vec-windows-x64@0.1.3: wanted {"os":"windows","cpu":"x64"} (current: {"os":"win32","cpu":"x64"})
npm ERR! notsup Valid os:   windows
npm ERR! notsup Actual os:  win32
npm ERR! notsup Valid cpu:  x64
npm ERR! notsup Actual cpu: x64
```

I don't know how the `sqlite-dist.toml` file is converted into npm packages, so I'm unable to create a PR for a fix unfortunately.

**Comment by mpdaugherty** [comment]
I think we may end up rewriting the node code from the node package so we can specify the location of the `.node` file explicitly and install it ourselves. It would be nicer to use an official package, however, which would stay up to date with the latest releases.

**Comment by asg017** [comment]
Hey @mpdaugherty, thanks for the report - can you try with the latest `0.1.4-alpha.2` version and let me know if it works?

The `os` field is now `win32`, which should solve this issue. If so I'll release a `0.1.4` with the same fix

**Comment by mpdaugherty** [comment]
Yes, thanks! I'll take a look, ideally can find some time today.

**Comment by asg017** [comment]
Thank you @mpdaugherty ! Will push out 0.1.4 shortly and will close this issue once it's out

**Comment by asg017** [comment]
Fixed in v0.1.4 and v0.1.6, Thank you again!
